### PR TITLE
Tag KernelEstimator v0.2.0

### PR DESCRIPTION
**Pull request by panlanfeng**
Hope this time is OK.
A new pull request to cover #7694.

### CODE REVIEW

**Comment by tkelman** [comment]
should remove the 0.1.8 files here

File: KernelEstimator/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,7 @@
+julia 0.5
+Distributions
+Optim
```
Comment (tkelman) [inline KernelEstimator/versions/0.2.0/requires]: what's the minimum Optim version requirement for https://github.com/panlanfeng/KernelEstimator.jl/pull/4 ? @pkofod

Comment (pkofod) [inline KernelEstimator/versions/0.2.0/requires]: That would be v0.5.0, as direct field access was used (we had deprecations on the functions for getting the results).

**Comment by panlanfeng** [comment]
Oops, I forgot to push the tag yesterday.
Do you know how to restart the checks?

**Pull request merged**